contracts/libraries/AccountLib.sol

```diff
@@ -173,10 +173,8 @@ function settle(
 ) returns (uint256 credited, uint256 debited, uint256 remainder) {
     int256 net = self.getNetBalance(token, account);
     if (net > 0) {
+        // Token remaining in internal balance or untracked tokens to transfer out.
         credited = uint256(net);
-        // unaccounted for tokens, e.g. transferred directly into Portfolio.
-        self.credit(msg.sender, token, uint256(net)); // gift to `msg.sender`.
-        self.reserves[token] += uint256(net); // add the difference back to reserves, so net is zero.
```

Comment:
No longer pays `msg.sender` for extra tokens (more physical balance than tracked reserves via a credit in the user's internal balance, which required the alignment with the `reserves. Expects `credited` to be transferred out in settlement.

Consequence: if any extra tokens in contract, they will remain extra until transferred out in `settlement`.